drivers/scheduler/k8s/specs/postgres-backup-with-suffix/px-postgres-app.yaml

```diff
@@ -0,0 +1,83 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: postgres-data-{{ .Suffix }}
```

Comment:
How about UID or something more generic? So I can use it any where.

Reply:
How about?

```yaml
metadata:
  {{- if .Suffix }}
  name: postgres-data-{{ .Suffix }}
  {{- else }}
  name: postgres-data-no-suffix
  {{- end }}
```

Reply:
Modified the spec.

Reply:
@kphalgun-px I included the conditional statement as mentioned, but the Torpedo CBT job fails and even the BYOC is failing. Can you please advise ?

Reply:
You have BYOC link?